Library/AppEnvironment.swift

```diff
@@ -133,9 +133,21 @@ public struct AppEnvironment: AppEnvironmentType {
       ubiquitousStore: next.ubiquitousStore,
       userDefaults: next.userDefaults
     )
+
+    // If there are no more items in the stack,
+    // then the next call to AppEvironment.current will fail.
+    assert(self.stack.count > 0)
```

Comment:
AFAIK we disable assertions for release builds. Plus, we already had a force unwrap when you call `AppEnvironment.current`, so the worst this would do would be to crash slightly sooner.